azure-pipelines.yml

```diff
@@ -176,7 +176,7 @@ jobs:
           pip install $_.FullName
       }
     displayName: 'Build NumPy'
-  - script: python runtests.py -n --show-build-log --mode=$(TEST_MODE) -- -rsx --junitxml=junit/test-results.xml
+  - script: python runtests.py --mode=$(TEST_MODE) -- -rsx --junitxml=junit/test-results.xml
```

Comment:
It is unfortunately common for environment variables not to persist between script stages in CI, so all the 32-bit adjustments in the previous stage are lost here.

Maybe try moving this modified line up into the previous `Build NumPy` stage and combine them to avoid duplicating the 32-bit env variable stuff again.

Reply:
If runtests.py can build NumPy, why are we using wheel and pip to build it before running runtests.py (same for MacOS)

Reply:
It is a better match to what happens for wheel builds on Windows at the moment for starters, and pip generally seems much more robust for Windows builds. I definitely like to see the dissonance reduced between the dev CI and wheels approach.

That said, I guess removing the `-n` does indeed appear to have fixed 1 of the 2 possible warnings for this PR (but not the one I thought).

Also, does removing `-n` mean that we're no longer testing the installed wheel, and instead rebuilding from source? I don't think we can do that then, so not sure how to handle the warning issue. The exercise did at least seem to prove that 1 of the warnings is sensitive to runtests.py involvement, and that your fix to distutils seems ok.